custom_components/alexa_media/config_flow.py

```diff
@@ -911,6 +917,11 @@ async def async_step_init(
             self.hass.config_entries.async_update_entry(
                 self.config_entry, data=user_input, options=self.config_entry.options
             )
+
+            self.hass.config_entries.async_update_entry(
```

Comment:
Why twice?

Reply:
An error!  Fixing...

Reply:
BTW, is it HA convention or your personal styling preference to have two blank lines preceding classes/functions as I'm noticing quite a bit of that??